src/System.Management.Automation/engine/Modules/NewModuleManifestCommand.cs

```diff
@@ -940,12 +940,12 @@ protected override void EndProcessing()
                 // Now open the output file...
                 PathUtils.MasterStreamOpen(
                     this,
-                    filePath,
+                    filePath,
 #if UNIX
-                    new UTF8Encoding(false), // UTF-8, no BOM
-#else
-                    EncodingConversion.Unicode, // UTF-16 with BOM
-#endif
+                    new UTF8Encoding(false), // UTF-8, no BOM
+#else
+                    EncodingConversion.Unicode, // UTF-16 with BOM
+#endif
```

Comment:
Is this part of change caused by some character encoding? If no intentional change was made, you can get rid of it by using `git reset HEAD^` and then `git add -p` (ignore this change section when adding).

Reply:
As part of https://github.com/PowerShell/PowerShell/pull/4761#issuecomment-327852126, this file has mixed EOL characters.  We can accept it to make the EOLs the same for this file or ignore it as I'm fixing this across the entire codebase in a separate PR.

Reply:
OK, then we can accept this part of the changes.